app/nexus/internal/initialization/recovery/recovery.go

```diff
@@ -260,7 +226,7 @@ func SendShardsPeriodically(source *workloadapi.X509Source) {
 //
 // The function first retrieves the root key from the system state. If no root
 // key exists, it returns an empty slice. Otherwise, it splits the root key into
-// shares using a secret sharing scheme, performs validation checks, and
+// shards using a secret sharing scheme, performs validation checks, and
```

Comment:
The docs were using "shares" and "shares" interchangeably; I tried to settle on "shard" as best as I can.

The only place I use "share" is `secretsharing.Share` shares, because the type "is" explicitly named "Share". -- Those shares, are "shards" in our nomenclature, but still, "share" makes it clearer there.